geode-core/src/main/java/org/apache/geode/internal/cache/BucketRegion.java

```diff
@@ -2544,4 +2544,9 @@ void superTxApplyPut(Operation putOp, Object key, Object wrappedNewValue, boolea
         versionTag, tailKey);
   }
 
+  @Override
+  Set<InternalDistributedMember> getDestroyRegionRecipients() {
+    return getSystem().getDistributionManager().getOtherDistributionManagerIds();
```

Comment:
Can we add comment to say why all distributed members are fetched...

Reply:
Comments added.